core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3221,7 +3203,7 @@ public String toString() {
   public static final PropertyKey USER_METRICS_HEARTBEAT_INTERVAL_MS =
       new Builder(Name.USER_METRICS_HEARTBEAT_INTERVAL_MS)
           .setAlias("alluxio.user.metrics.heartbeat.interval.ms")
-          .setDefaultValue("1min")
+          .setDefaultValue("10sec")
```

Comment:
It's hard to choose the default value for this interval. 
We want it to be big enough to make sure master can handle that many client connections. --> reduce client connections overhead on the leading master.
Also we want it to be small enough that we can see more updated changes in the leading master metrics page/sink.

So it will be good to choose the smallest number under the criteria that leading master can handles.
I expect that we can handle 1000 workers with heartbeat 1sec to the leading master. Then I feel we are likely to handle 10,000 workers with heartbeat interval 10sec.